### ISSUE TITLE
Proposal of Distribution of funds through a Simple DAO governed faucet. 

### ISSUE THREAD

**Issue by glauseWilde** [gitcoinco/ERC-1789#41]
I have been following along the conversation and want to present a plan for collection and dispersal of funds. Thus far the only other formal proposal of what to do with the funds once collected has been Evan Van Ness's. That is that stewardship be directly under the EF. I offer this as an alternative.

![Ethereum Block Dev Fund Mechanism 1 4 19](https://user-images.githubusercontent.com/49164280/55356589-01f99300-54c3-11e9-8ea1-bbb736a63d80.png)

I want to clarify that in my eyes it is best if those who benefit from the system take part in the pain of providing for that system. This is my argument towards any funding being an addition to the existing block reward rather than be taken from the miners share.

Members of the meta-dao are welcome to be made from trusted members of the EF as well as other trusted community members. I have nothing against the EF. Overall transparency within the system is my highest priority. 

Feedback appreciated, 
Glause A. Wilde

Thank you to @lrettig , @owocki, jpitts and madeoftin for initial inspiration.

**Comment by owocki** [comment]
Glause this is awesome!

from here, i think it'd be cool to be able to see visually see the accountability mechanism.  after funding group a and funding group b are fully funded, there is a governance mechanism in which the meta-DAO will go back to the core devs / community and lobby for an additional round of block rewards funding. so there is a continual accountability mechanism for the metadao / funding groups in this form.

**Comment by glauseWilde** [comment]
@owocki Added in a more complete life-cycle view. Better?

It is getting a bit wider then I anticipated, but overall looking fairly good. Anyone have feedback on things that are missing?

 * Adding an accountability mechanism for the fund groups is on the Todo List

![Ethereum Block Dev Fund Mechanism 2 4 19](https://user-images.githubusercontent.com/49164280/55441783-1187e300-557a-11e9-9f58-8405b4ae1049.png)

**Comment by owocki** [comment]
this is dope thank you @glauseWilde 

one thing i'm thikning we might want to add is how individual funding groups are kept accountable.  basically the funding groups are going to want to jockey for higher percentage of "meta-dao funds", and in the the accountability phase their performance (in addition to the meta-dao's performance) will be important to gauge

**Comment by jpitts** [comment]
Really nice. We should encourage the use of diagrams to discuss proposals!

Some additional functions of the Meta-DAO might be:
- Stating the general funding priorities (so that funding group behavior can be guided)
- Defining tranches (releasing funds in chunks as goals are met)

**Comment by jpitts** [comment]
@glauseWilde, is it proposed here that the "funding groups" would make the more fine-grained allocation decisions after receiving funds from the Meta-DAO? If that is the case, tranches and other controls could be at the funding group level.

**Comment by GregTheGreek** [comment]
If I understand correctly, Funding Groups A & B are "organizations" in charge of then disbursing the funds?

**Comment by glauseWilde** [comment]
> @glauseWilde, is it proposed here that the "funding groups" would make the more fine-grained allocation decisions after receiving funds from the Meta-DAO? If that is the case, tranches and other controls could be at the funding group level.

Yes that is the intent. The funding groups would be the ideal place for that. My hope is that the meta-dao have no custody over funds, only direction of the faucet. The funding groups would accumulate funds through the faucet, and as they perform it can be removed or increased. A layer 1 and layer 2 approach if you will :)

**Comment by alexvandesande** [comment]
This is not very clear for me. Are funds then controlled by the Meta DAO? And how is the MetaDAO controlled?

**Comment by glauseWilde** [comment]
> If I understand correctly, Funding Groups A & B are "organizations" in charge of then disbursing the funds?

There also could be more or less number of groups. The meta-dao controls the number of groups receiving funding and the amount of the faucet they receive. The meta-dao has a hard limit on what the cumulative total faucet (lets say .1 Eth per block for example), but they aren't forced to always give out all of the faucet. 

An Example: one could be a developer Dao ` .02Eth `, gitcoin grants ` .02Eth `, and a subgroup in the EF ` .02Eth `. ` Total of .06 of the faucet being released the rest being burned ` 

And lets say the Dev dao is a dud and so the meta-dao shuts off their faucet ` now giving out a total of .04Eth of the faucet ` . Later on a different group shows up that the meta-DAO sees as worth trying. They then can be given their own allocation ` .01Eth `  ` total allocation is currently .05 `

This gives room for groups that do well to get more, and to ramp up slowly.

**Comment by glauseWilde** [comment]
> This is not very clear for me. Are funds then controlled by the Meta DAO? And how is the MetaDAO controlled?

The metaDao controls the faucet and membership in the metaDao. That is all they do. They don't have a pot of accumulated funds they doll out.

**Comment by glauseWilde** [comment]
> this is dope thank you @glauseWilde
> one thing i'm thikning we might want to add is how individual funding groups are kept accountable.  basically the funding groups are going to want to jockey for higher percentage of "meta-dao funds", and in the the accountability phase their performance (in addition to the meta-dao's performance) will be important to gauge

I see it as two cycles of accountability. One larger cycle where the meta-dao with the core-devs and the community that has something like an 18 month timeline.

Inside of that is a smaller near real time (daily or weekly) cycle of accountability from the funding groups to the meta-DAO. Allocations of the faucet can be changed dynamically within the 18th month timeline.

**Comment by alexvandesande** [comment]
Everything is on chain, right? I think what you meant by "On-off-chain" is actually "consensus layer".

So basically you have a meta-dao which decides the distribution of the block rewards. I would suggest 2 things to add here:

* There should be a hard time delay between the changes being decided by the meta-dao and then the rewards actually changing. The delay should be long enough for the community to organize a hard fork if they wanted to rebel against the decision, maybe something like 4-9 weeks.
* Membership of the Metadao should not be with individuals, but other DAOs that themselves could represent different "stakeholders" and have their own internal governance. This would enable more complex governance layers to be designed: futarchy, quadratic voting, hash voting, etc, all would be implemented not on the meta-dao level, but on the member level

**Comment by glauseWilde** [comment]
> Everything is on chain, right? I think what you meant by "On-off-chain" is actually "consensus layer".

That there is a faucet and the amount to be disbursed and the emission schedule is on chain. Where the faucet is delivered ultimately would be decided off chain transparently. Here are some relevant comments from @lrettig about tweaking relative per-DAO(funding groups in this case) allocations more frequently [comment 1]( https://github.com/ethereum-funding/blockrewardsfunding/issues/25#issuecomment-476786938) [comment 2](https://github.com/ethereum-funding/blockrewardsfunding/issues/25#issuecomment-477417586l)

> So basically you have a meta-dao which decides the distribution of the block rewards. I would suggest 2 things to add here:
> 
> There should be a hard time delay between the changes being decided by the meta-dao and then the rewards actually changing. The delay should be long enough for the community to organize a hard fork if they wanted to rebel against the decision, maybe something like 4-9 weeks.

Again to Lane's point plus decision fatigue among the community. The meta-dao is given a time allotted (around 18 months would be good in my opinion) to do as they feel fulfills their mandate. This would be the deal presented to the community and if they accept then it goes through. If they mess up during the dispersion phase they won't get a second chance. If they do well, then the community can consider another round of block rewards.

> Membership of the Metadao should not be with individuals, but other DAOs that themselves could represent different "stakeholders" and have their own internal governance. This would enable more complex governance layers to be designed: futarchy, quadratic voting, hash voting, etc, all would be implemented not on the meta-dao level, but on the member level

All of these things could be represented within a Funding Group. A funding group could use a DAO with  " futarchy, quadratic voting, hash voting, etc." as you say. The metaDAO being as simple as possible gives space for greater complexity for the funding groups, or the layer2 groups (co-opting some blockchain terminology). If part of the metaDAO fails it can risk halting the entire system. Which to me means the simpler the lower level mechanism the better. Also, increasing time to build it means waiting longer before funds can be distributed when funds for network development are needed today.

I can see there are a lot of amazing experiments being done on consensus and DAOs. I would much rather have a system that enables greater experimentation and can adapt to the ever shifting landscape of blockchain innovation.

**Comment by glauseWilde** [comment]
Hell, an individual could step up and say "I will vote on any motion the way x dao votes". If the metaDAO members see value in it he could very well be voted in.

**Comment by alexvandesande** [comment]
> The meta-dao is given a time allotted (around 18 months would be good in my opinion) to do as they feel fulfills their mandate.

Ah so this would work only for 18 months and then would require another hard fork to reactivate?

**Comment by glauseWilde** [comment]
> Ah so this would work only for 18 months and then would require another hard fork to reactivate?

Exactly

**Comment by lirazsiri** [comment]
A policeman sees a drunk man searching for something under a streetlight and asks what the drunk has lost. He says he lost his keys and they both look under the streetlight together. After a few minutes the policeman asks if he is sure he lost them here, and the drunk replies, no, and that he lost them in the park. The policeman asks why he is searching here, and the drunk replies, "this is where the light is"

This joke is from the Wikipedia article on the streetlight effect, which this discussion on DAO governed faucets is a perfect example of: https://en.wikipedia.org/wiki/Streetlight_effect

"The streetlight effect, or the drunkard's search principle, is a type of observational bias that occurs when people only search for something where it is easiest to look"

In other words, maybe we shouldn't be spending so much attention on who gets the funds but on how funds from the inflation treasury are used to maximize positive impacts for Ethereum. Figuring that out is the hard problem. Coming up with the technical mechanism for distributing funds to a self selected club of cool kids is the easy problem.

**Comment by glauseWilde** [comment]
> In other words, maybe we shouldn't be spending so much attention on who gets the funds but on how funds from the inflation treasury are used to maximize positive impacts for Ethereum

This is the job for the members of the multi-sig. Hats off to those members because I do not plan on being one of them. Funding groups are accountable to the multSig members to this idea. The multi-sig members accountable to Ethereum as whole. 

In fact we should add this as a directive for the metaDao and the funding groups

> Use the block rewards to maximize positive impacts for Ethereum Ecosystem development

**Comment by owocki** [comment]
@glauseWilde  what is your twitter handle? i want to give u credit

https://mobile.twitter.com/owocki/status/1115407139864842240
